score-client/Dockerfile

```diff
@@ -8,7 +8,7 @@
 # Banner @ https://goo.gl/Yyoc6R
 
 FROM       ubuntu:16.04
-MAINTAINER ICGC <dcc-support@icgc.org>
+MAINTAINER SCORe <>
```

Comment:
Just remove the MAINTAINER statement, this is deprecated in docker: https://docs.docker.com/reference/dockerfile/#maintainer-deprecated